scripts/lib-franklin.js

```diff
@@ -123,6 +123,18 @@ export function toCamelCase(name) {
   return toClassName(name).replace(/-([a-z])/g, (g) => g[1].toUpperCase());
 }
 
+/**
+ * Returns a Franklin icon span (that will be expanded by decorateIcons)
+ * @param {string} name The icon file name (minus ".svg")
+ * @returns {HTMLSpanElement}
+ */
+export function createIcon(name) {
```

Comment:
AFAIK global custom code should not be placed in `lib-franklin.js` but rather in `scripts.js`, as `lib-franklin` is basically the versioned framework code.

Reply:
Fixed